.travis.yml

```diff
@@ -96,12 +87,18 @@ before_script:
 script:
   - ./travis_configure.sh run_tests
 
-# Deploy debian packages to bintray when building for a git tag
+## Final clean up scripts
+after_script:
+  - ./travis_configure.sh after_script
+
+## Deploy debian packages to bintray
+##
+## deb package built on ubuntu xenial with gcc
 deploy:
     provider: script
-    script: beaver bintray upload -N pkg/deb/*.deb
+    script: beaver bintray upload -o -N pkg/deb/*.deb
     skip_cleanup: true
     on:
-        tags: true
-        repo: sociomantic-tsunami/libdrizzle-redux
-        condition: $TRAVIS_OS_NAME = 'linux' && $CC = 'gcc' && $DIST_PACKAGE_TARGET = 'DEB'
+        tags: true # must be a git tag
+        repo: sociomantic-tsunami/libdrizzle-redux # must be upstream
+        condition: $TRAVIS_OS_NAME.$DIST_NAME.$DIST_VERSION = 'linux.ubuntu.xenial' && $CC = 'gcc' && $MAKE_TARGET =~ 'deb' && $DEPLOY_PACKAGE == 'true'
```

Comment:
Why concatenating `$TRAVIS_OS_NAME.$DIST_NAME.$DIST_VERSION` but not `$CC`, `$MAKE_TARGET` and `$DEPLOY_PACKAGE`?

Reply:
Yes (with `=`, not `=~`). Actually I think is safer to compare every string individually, as if you have weird combinations the concatenation could match when the individual parts wont. I think is unlikely enough to be acceptable, but I really don't see any reason not to compare every variable individually instead.